invenio_rdm_records/theme/templates/invenio_rdm_records/record_landing_page.html

```diff
@@ -30,7 +30,9 @@
             <div class="left floated left aligned column">
               <span class="ui label teal"
```

Comment:
Can you also remove the label around the publication date. I think there's an issue for it reported.